IanGRap/Squish
# adjust ControllerSignIn.cs to not allow changing hats while the player is ready

Contributors: Alfredo

Changes: 
* ControllerSignIn.cs: Added new checks to make sure the player is not ready when they try changing hats or skins

How to test: 
* Go into lobby
* ready up on of the players
* try changing hats and skins. You should only be able to when the player does not have a check mark

PS. This kind of made me realize that the ControllerSignIn.cs needs to be retouched a bit. There are places where code could be put into functions
